tests/ci_build/deploy/ci_deploy_doc.sh

```diff
@@ -26,10 +26,7 @@
 #
 # BUILD_ID: the current build ID for the specified PR
 #
+set -ex
```

Comment:
Do you need set +ex at the end to disable?

Reply:
No, this is restricted to the current scope (aka this script).

Reply:
Cool. As long as the script is not executed through `source` it's fine.